docs/source/reference.md

```diff
@@ -0,0 +1,63 @@
+# Reference
+
+## JSON configuration overview
+
+This is an auto-generated overview of all the possible settings each `menuinst` JSON file can include.
+Note that each platform sub-dict can override top-level values if required.
+See Schema below for more details.
+
+{{ default_schema_json }}
+
+(schema)=
+
+## Configuration schema
+
+```{eval-rst}
+.. autopydantic_model:: menuinst._schema.MenuInstSchema
+.. autopydantic_model:: menuinst._schema.MenuItem
+.. autopydantic_model:: menuinst._schema.Platforms
+.. autopydantic_model:: menuinst._schema.Linux
+.. autopydantic_model:: menuinst._schema.MacOS
+    :member-order: groupwise
+
+.. autopydantic_model:: menuinst._schema.Windows
+```
+
+(placeholders)=
+
+## Placeholders
+
+The JSON configuration files support several placeholders if surrounded with spaced double curly braces: `{{ variable }}`.
+Note these are _not_ Jinja templates. It just follows the same syntax, but accepts no transformations. Only variable names are accepted!
```

Comment:
We are saying no jinja, so maybe a link to jinja templates or more info in case the user wants to know more?

Also we are using "variable names" which sounds a bit also like a jinja concept, so might be useful to link to any info on this?

Reply:
Rephrased a bit for clarity.